### ISSUE TITLE
import {} does not work with this package.

### ISSUE THREAD

**Issue by DoormatIka** [appy-one/acebase#203]
The error is:
`error TS2305: Module '"acebase"' has no exported member 'AceBase'.`
Code: 
`import { AceBase } from "acebase";`
A band-aid to this is changing the `index.d.ts` file from
```
export * from '../types;
to
export * from '../types/index.js';
```

package.json
```
{
  "name": "kogasa-ts",
  ...
  "license": "ISC",
  "type": "module",
  "dependencies": {
    "acebase": "^1.27.0",
    "discord.js": "^14.7.1",
    "dotenv": "^16.0.3",
    "openai": "^3.1.0"
  }
}
```

**Comment by appy-one** [comment]
What is you typescript version?

**Comment by appy-one** [comment]
Closing this issue because there has been no response

[![Sponsor AceBase](https://user-images.githubusercontent.com/26569719/168233053-8e56b243-4140-40ab-9a30-4cb3cc149bfe.svg)](https://github.com/sponsors/appy-one) [![Spread the word](https://user-images.githubusercontent.com/26569719/169265089-3d593555-e1ad-4390-986b-877ac2c38a47.svg)](https://twitter.com/intent/tweet?button=&url=https://github.com/appy-one/acebase&text=I'm+using+@AcebaseRealtime+in+my+project+to+make+my+life+easier!&button=)  [![contribute](https://user-images.githubusercontent.com/26569719/169265318-30c4c6a5-7c89-46a0-a7a2-ef433a8192f4.svg)](https://github.com/appy-one/acebase#contributing)

**Comment by appy-one** [comment]
This has been fixed in v1.28.3

[![Sponsor AceBase](https://user-images.githubusercontent.com/26569719/168233053-8e56b243-4140-40ab-9a30-4cb3cc149bfe.svg)](https://github.com/sponsors/appy-one) [![Spread the word](https://user-images.githubusercontent.com/26569719/169265089-3d593555-e1ad-4390-986b-877ac2c38a47.svg)](https://twitter.com/intent/tweet?button=&url=https://github.com/appy-one/acebase&text=I'm+using+@AcebaseRealtime+in+my+project+to+make+my+life+easier!&button=)  [![contribute](https://user-images.githubusercontent.com/26569719/169265318-30c4c6a5-7c89-46a0-a7a2-ef433a8192f4.svg)](https://github.com/appy-one/acebase#contributing)
